### PR TITLE
Solution to Week1 Task

### PR DESCRIPTION
**Pull request by Umaynah**
THIS IS A PULL REQUEST TEMPLATE! This helps you complete a pull request. Please follow the instructions below. 

## Description
Tell us about your pull request!

**Please make sure your submission is in the correct directory**. In most cases, this is in submissions/YOUR-GITHUB-USERNAME/Week-Number/filename.

For example, if @codedentwickler is completing Week 1:
- I solved the Week 1 task.
- I create a new file inside submissions/codedentwickler/Week1/
- I name this file fizzbuzz.py, hence we have the fizzbuzz solution in submissions/codedentwickler/Week1/fizzbuzz.py

### CODE REVIEW

**Comment by Umaynah** [comment]
I created the sub-folder "Week1"
I uploaded my solution to the FizzBuzz test into the Week1 sub-folder
I also have a new file "about me.md" where you can get to know me.

File: submissions/Umaynah/Week1/Fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#RAJI HUMAINAT TIMILEHIN
+#FizzBuzz Online Test
+
```
Comment (codedentwickler) [inline submissions/Umaynah/Week1/Fizzbuzz.py:3]: Nice solution.

File: submissions/Umaynah/about.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+I am Humainah, an Electrical and Electronics Engineering student of UI. I love to write and code.
+
+
```
Comment (codedentwickler) [inline submissions/Umaynah/about.md]: Kindly remove unnecessary lines. One line seems to be enough

**Review by codedentwickler — CHANGES_REQUESTED** [review]
Minor comments to attend to.

**Comment by Umaynah** [comment]
Edited.

File: submissions/Umaynah/Week1/Fizzbuzz.py (hunk 1 above)
Comment (Umaynah) [reply submissions/Umaynah/Week1/Fizzbuzz.py:3]: Thanks for the review

**Review by codedentwickler — APPROVED** [review]

**Pull request merged**